### PR TITLE
🚧Update org contribution doc

### PR DESCRIPTION
**Pull request by CalvinWilkinson**
<h1 style="font-weight:bold" align="center">Feature Pull Request</h1>
<h2 style="font-weight:bold" align="center">📋Review Checklist📋</h2>

- [x] The **_head(source)_** branch for this pull request is a **_feature_** branch, with the correct naming syntax, in the following example:
  - A **_head(source)_** branch is the branch being merged into a target branch.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: feature/<issue-num>-<description>
      Example: feature/123-my-feature
      ```
    </details>

- [x] The **_head(source)_** branch for this pull request is created from a **_development_** branch with the name **_develop_**.

- [x] The **_base(target)_** branch for this pull request is a **_development_** branch with the name **_develop_**.
  - A **_base(target)_** branch is the branch that the **_head(source)_** branch is merging into.

💡For more information on branching, refer to the project's [branching documentation](https://github.com/KinsonDigital/.github/blob/master/docs/BranchingFullGitFlow.md).

- [x] The associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [x] The pull request title matches the title of the associated issue.

- [x] The labels attached to this PR match the labels attached to the associated issue.

- [x] The assigned project matches the project of the associated issue.

- [x] The assigned milestone matches the milestone of the associated issue.

- [x] My code follows the coding style of this project.
  - The style is enforced by the **_.editorconfig_** files in the project and displayed as warnings.  If there is an edge case, reach out, and let's discuss it.

- [x] I have manually tested my code changes to the best of my ability.

- [x] All tests passed locally.
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
  - Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
  - Status checks run for this pull request when merging into the **_develop_** branch.  These status checks run every time a change is pushed to the pull request.  These checks validate version syntax, tagging, builds, unit tests, and more.

### CODE REVIEW

**Review by KinsonDigitalAdmin — APPROVED** [review]

File: docs/CreatingModel1Branches.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+     ```cli
+     git fetch upstream
+     git checkout develop
+     git merge upstream/develop
+     git push
+     git checkout preview/v1.2.3-preview.4
+     ```
+
+   - If implementing a <span style="color: #9E269E;font-weight:bold">preview feature</span>  
+     - If the <span style="color: #CC99FF;font-weight:bold">preview</span> branch already exists locally and in your fork.
```
Comment (kselena) [inline docs/CreatingModel1Branches.md:17]: Reword
```suggestion
   - If implementing a <span style="color: #9E269E;font-weight:bold">preview feature</span> when the <span style="color: #CC99FF;font-weight:bold">preview</span> branch already exists locally and in your fork.
```

File: docs/CreatingModel1Branches.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+         ```cli
+         git fetch upstream
+         git checkout preview/v1.2.3-preview.4
+         git merge upstream/preview/v1.2.3-preview.4
+         git push
+         git checkout preview/v1.2.3-preview.4
+         ```
+
+     - If the <span style="color: #CC99FF;font-weight:bold">preview</span> branch has never been checked out from the _**upstream**_ repository before, this means it does not exist in your fork.  Use the commands below to update your fork with the new branch from the _**upstream**_ repository.
```
Comment (kselena) [inline docs/CreatingModel1Branches.md:27]: Reword
```suggestion
     - If the <span style="color: #CC99FF;font-weight:bold">preview</span> branch has never been checked out from the _**upstream**_ repository, it means that it doesn't exist in your fork.  Use the commands below to update your fork with the new branch from the _**upstream**_ repository.
```

File: docs/CreatingModel1Branches.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+       ```cli
+       git fetch upstream
+       git checkout -b preview/v1.2.3-preview.4 upstream/preview/v1.2.3-preview.4
+       git push -u origin preview/v1.2.3-preview.4
+       ```
+
+### **2. Create a Working Branch**
+
+Create a <span style="color: #66B2FF;font-weight:bold">feature</span> or  branch using the syntax requirements in the _**[Branching Docs](https://github.com/KinsonDigital/.github/blob/master/docs/Branching.md)**_ and push it to the fork remote. 
+   - Creating a <span style="color: #66B2FF;font-weight:bold">feature</span> branch. Make sure the <span style="color: #FFB366;font-weight:bold">develop</span> branch is checked out.
```
Comment (kselena) [inline docs/CreatingModel1Branches.md:38]: Reword
```suggestion
   - Make sure the <span style="color: #FFB366;font-weight:bold">develop</span> branch is checked out when creating a <span style="color: #66B2FF;font-weight:bold">feature</span> branch. 
```

File: docs/CreatingModel1Branches.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+      ```cli
+      git checkout -b feature/123-my-branch
+      git push --set-upstream origin feature/123-my-branch
+      ```
+
+   - Creating a  branch. Make sure the <span style="color: #CC99FF;font-weight:bold">preview</span> branch is checked out.
```
Comment (kselena) [inline docs/CreatingModel1Branches.md:45]: Reword
```suggestion
   - Make sure the <span style="color: #CC99FF;font-weight:bold">preview</span> branch is checked out when creating a  branch. 
```

File: docs/CreatingModel1Branches.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+      ```cli
+      git checkout -b preview/feature/123-my-branch
+      git push --set-upstream origin preview/feature/123-my-branch
+      ```
+
+<h3 style="border:0;font-weight:bold" align="center">Additional Info</h3>
+
+- Please always sync changes from the _**upstream**_ to your fork before creating your branch.
+- Please make sure that you create your <span style="color: #66B2FF;font-weight:bold">feature</span> or  branch from the correct branch.
+- If you are creating a  branch and you do not see the <span style="color: #CC99FF;font-weight:bold">preview</span> branch to create it from, look for the branch in the _**upstream**_ and check out the branch and push it to your fork so your fork has a copy.
```
Comment (kselena) [inline docs/CreatingModel1Branches.md:56]: Reword
```suggestion
- If you are creating a branch and you do not see the <span style="color: #CC99FF;font-weight:bold">preview</span> branch to create it from, look for the branch in the _**upstream**_ ,check out the branch and push it to your fork as a copy.
```

File: docs/EarlyPullRequests.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+2. If you have not already done so, complete the **Create a Branch to Contribute** section above.
+3. Create an empty commit.  This allows you to create a PR without any changes.
+   ```cli
+   git commit --allow-empty -m "Start work for issue #123"
+   git push
+   ```
+4. Create a [**draft pull request**](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests). Do this **BEFORE** you start working.
+   > **Note**
+   Please make sure that the title of the PR _**EXACTLY**_ matches the title of the associated issue.
+   > - Use one of the PR templates below depending on if you are doing a regular feature or a preview feature.
```
Comment (kselena) [inline docs/EarlyPullRequests.md:13]:
```suggestion
   Please make sure that the title of the PR _**EXACTLY**_ matches the title of the associated issue (emoji's included).
   > - Use one of the PR templates below depending whether you are doing a regular feature or a preview feature.
```

File: docs/HowToFork.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
+<h2 style="border:0;font-weight:bold" align="center">How to Fork the Repository</h2>
+
+### **1. Fork the repository**
+First, fork the repository.
```
Comment (kselena) [inline docs/HowToFork.md:4]: Remove this line, it's repetitive
```suggestion
First, fork the repository.
```

File: docs/HowToFork.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+
+> **Note**
+> For more information on how to fork a repository, go [here](https://docs.github.com/en/get-started/quickstart/fork-a-repo).
+
+### **2. Clone the forked repository**
+Clone the forked repository to your machine.
```
Comment (kselena) [inline docs/HowToFork.md:10]: Remove this line, it's repetitive 
```suggestion
Clone the forked repository to your machine.
```

File: docs/HowToFork.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+> **Note** Swap `johndoe` with your username and `myproject` with the name of the project to clone.
```
Comment (kselena) [inline docs/HowToFork.md:11]: Reword
```suggestion
> **Note** When cloning the repo to your machine, be sure to swap `johndoe` with your username and `myproject` with the name of the project to clone.
```

File: docs/HowToFork.md [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+   ```cli
+   git clone https://github.com/johndoe/myproject.git
+   cd myproject
+   git remote add upstream https://github.com/KinsonDigital/myproject.git
+   ```
+
+### **3. Let GIT know who you are**
+To better track changes and who does what, it's a good practice to give GIT some information about yourself.
```
Comment (kselena) [inline docs/HowToFork.md:19]: Reword
```suggestion
To better track changes and know who did what, it's good practice to give GIT some information about yourself.
```

**Review by kselena — CHANGES_REQUESTED** [review]

File: docs/EarlyPullRequests.md (hunk 6 above)
Comment (CalvinWilkinson) [reply docs/EarlyPullRequests.md:13]: I do not want to call out the emoji thing since the templates already do.  If for some reason in the future, we decide to get rid of emojis, I don't want to have to update this document.  Copy the title example means the emojis for the most part anyways.

File: docs/EarlyPullRequests.md [hunk 11]
```diff
@@ -10,7 +10,7 @@
 4. Create a [**draft pull request**](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests). Do this **BEFORE** you start working.
    > **Note**
    Please make sure that the title of the PR _**EXACTLY**_ matches the title of the associated issue.
-   > - Use one of the PR templates below depending on if you are doing a regular feature or a preview feature.
+   > - Use one of the PR templates below depending on whether if you are doing a regular feature or a preview feature.
```
Comment (kselena) [inline docs/EarlyPullRequests.md:13]: Remove 'if'
```suggestion
   > - Use one of the PR templates below depending on whether you are doing a regular feature or a preview feature.
```

**Review by kselena — CHANGES_REQUESTED** [review]

**Review by kselena — APPROVED** [review]

**Review by KinsonDigitalAdmin — APPROVED** [review]

**Review by kselena — APPROVED** [review]

**Review by kselena — APPROVED** [review]
